src/java/test/org/apache/zookeeper/server/ServerStatsTest.java

```diff
@@ -73,11 +73,11 @@ public void testLatencyMetrics() {
 
         // Then ...
         assertThat("Max latency check", 2000L,
-                greaterThanOrEqualTo(serverStats.getMaxLatency()));
+                lessThanOrEqualTo(serverStats.getMaxLatency()));
```

Comment:
Thanks for catching this and fixing it! The serverstats.getMaxLatency() should of went in the "actual" parameter and the magic number as the greaterThanOrEqualTo parameter. (I did this test. Sorry...)
But changing greater to less work just fine.